crates/contract/src/lib.rs

```diff
@@ -1601,7 +1603,7 @@ impl MpcContract {
 }
 
 fn try_state_read<T: borsh::BorshDeserialize>() -> Result<Option<T>, std::io::Error> {
-    env::storage_read(b"state")
+    env::storage_read(b"STATE")
```

Comment:
ups yeah I had seen it as well, so shame on me

Reply:
Shhhh :hushed: